Server/Plugins/APIDump/Hooks/OnPlayerEditingBook.lua

```diff
@@ -0,0 +1,28 @@
+return
+{
+	HOOK_PLAYER_EDITING_BOOK =
+	{
+		CalledWhen = "A player is editing a book.",
+		DefaultFnName = "OnPlayerEditingBook",  -- also used as pagename
+		Desc = [[
+			This hook is called whenever a {{cPlayer|player}} is editing a book.
+			See also the {{OnPlayerEditingBook|HOOK_PLAYER_EDITED_BOOK}} hook for a similar hook, is called when a
+			player has edited a book.
```

Comment:
Add the note that the plugin may override the player edits by changing the NewContent parameter